src/js/components/organisms-page/resellers/resellers.js

```diff
@@ -32,6 +43,41 @@ export default {
         },
         showAllAuthorizedResellers() {
             this.visibleAllAuthorizedResellers = true;
+        },
+        newRow(el) {
+            const rect = el.getBoundingClientRect(),
+                index = el.dataset.index - this.showMax;
+
+            cardsYPos.push(rect.top);
+
+            if (cardsYPos.length > 1) {
+                return cardsYPos[index - 1] !== cardsYPos[index];
+            }
+
+            return false;
+        },
+        setDelay(el) {
```

Comment:
This should be calculateAnimationDelay, because the function doesn't set something :-)